openpype/hosts/maya/plugins/publish/collect_arnold_scene_source.py

```diff
@@ -1,35 +1,31 @@
 from maya import cmds
-from openpype.pipeline.publish import KnownPublishError
 
 import pyblish.api
 
 
-class CollectAssData(pyblish.api.InstancePlugin):
-    """Collect Ass data."""
+class CollectArnoldSceneSource(pyblish.api.InstancePlugin):
+    """Collect Arnold Scene Source data."""
 
     # Offset to be after renderable camera collection.
     order = pyblish.api.CollectorOrder + 0.2
-    label = 'Collect Ass'
+    label = "Collect Arnold Scene Source"
     families = ["ass"]
 
     def process(self, instance):
-        objsets = instance.data['setMembers']
+        objsets = instance.data["setMembers"]
 
         for objset in objsets:
             objset = str(objset)
             members = cmds.sets(objset, query=True)
             if members is None:
                 self.log.warning("Skipped empty instance: \"%s\" " % objset)
                 continue
-            if "content_SET" in objset:
-                instance.data['setMembers'] = members
-                self.log.debug('content members: {}'.format(members))
-            elif objset.startswith("proxy_SET"):
-                if len(members) != 1:
-                    msg = "You have multiple proxy meshes, please only use one"
-                    raise KnownPublishError(msg)
-                instance.data['proxy'] = members
-                self.log.debug('proxy members: {}'.format(members))
+            if objset.endswith("content_SET"):
```

Comment:
What logic is changing?

Reply:
We used to check if "content_SET" was in the string (in any position) and now we just check if it ends with that.

Reply:
Ahh, yeah. This is because if you want to have multiple ASS instances in the scene, we need to prefix the objectsets else they will end up with a number in the name cause of the duplicate names in the scene. Cant have objectsets with the same names even if they are nested.